src/schema/basic_classes.yaml

```diff
@@ -305,7 +305,7 @@ classes:
         structured_pattern:
           syntax: "^{id_nmdc_prefix}:(wfmag|wfmb|wfmgan|wfmgas|wfmsa|wfmp|wfmt|wfmtan|wfmtas|wfnom|wfrbt|wfrqc)-{id_shoulder}-{id_blade}{id_version}$|^{id_nmdc_prefix}:(omprc|dgms|dgns)-{id_shoulder}-{id_blade}$"
           interpolated: true
-
+  
```

Comment:
I think the addition of this space was unintentional. Committing this suggestion will remove it, simplifying the PR diff.

```suggestion

```